HEP/src/components/GlobalTextFields.tsx

```diff
@@ -0,0 +1,23 @@
+import { createElement } from "typed-html";
+import { SectionDataElement, CategoryOptionCombo } from "../models/Form";
+
+interface GlobalTextFieldAttributes {
+    dataElements: SectionDataElement[];
+    categoryOptionCombos: CategoryOptionCombo[];
+}
+
+export function GlobalTextFields(attributes: GlobalTextFieldAttributes): string {
+    const { dataElements, categoryOptionCombos } = attributes;
+    const fields = dataElements.map(de =>
+        categoryOptionCombos.map(coc => [
```

Comment:
`map` inside a `map`, this creates an array of arrays. We can do a _.flatMap or _.flatten just before using it (now I see that's what you are doing in the rest of the code, so this one)